### PR TITLE
Add core members feature - Delivers #134666421

### PR DESCRIPTION
**Pull request by susannasouv**

### CODE REVIEW

File: app/helpers/core_members_helper.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module CoreMembersHelper
+  def reset_is_core_member
+    users = User.all
+    for user in users
+      user.update_attributes({is_core_member: false})
+    end
+  end
+
+  def set_is_core_member(user_ids)
+    for user_id in user_ids
```
Comment (jeremyjpark) [inline app/helpers/core_members_helper.rb]: A map here might be a good idea

File: app/views/core_members/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <th>
+      Email
+    </th>
+  </tr>
+  </thead>
+  <tbody>
+  <% if @core_members.empty? %>
+          <tr><td>It looks like there are no Core Members yet!</td></tr>
+  <% else %>
+      <% @core_members.each do |core_member| %>
```
Comment (jeremyjpark) [inline app/views/core_members/index.html.erb:18]: Might be worth writing a function to output some of the html in the model?

**Review by aganpule — APPROVED** [review]
lgtm

**Review by jeremyjpark — APPROVED** [review]

**Review by LizzySteger — APPROVED** [review]
Looks good to me!

**Pull request merged**